src/plugins/legacy/polygonRoi/toolboxes/contoursManagementToolBox.cpp

```diff
@@ -636,47 +583,23 @@ void contoursManagementToolBox::addLabel()
     if (specialities->currentText()==urologySpeciality
             && specialities->currentIndex()==1)
     {
-        if (targetButton->value())
```

Comment:
question to @juliencastelneau : default target value that we want is false or true?